vr-wordle/src/App.js

```diff
@@ -176,7 +170,7 @@ export default function App() {
         fade
       />
       <fog attach="fog" args={["#421700", 0, 100]} />
-      <PositionalAudio url="/wind.mp3" distance={1} autoplay loop />
+      <Wind />
     </VRCanvas>
```

Comment:
What does this Wind component do?

Reply:
Adds wind sound effects to the scene, I wanted to componentize as much as possible